temporal-sdk/src/main/java/io/temporal/client/WorkflowExecutionDescription.java

```diff
@@ -21,19 +21,60 @@
 package io.temporal.client;
 
 import io.temporal.api.workflowservice.v1.DescribeWorkflowExecutionResponse;
+import io.temporal.common.Experimental;
 import io.temporal.common.converter.DataConverter;
+import io.temporal.payload.context.WorkflowSerializationContext;
 import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 
 /** Contains information about a workflow execution. */
 public class WorkflowExecutionDescription extends WorkflowExecutionMetadata {
+  private final @Nonnull DataConverter dataConverter;
   private final @Nonnull DescribeWorkflowExecutionResponse response;
 
   public WorkflowExecutionDescription(
       @Nonnull DescribeWorkflowExecutionResponse response, @Nonnull DataConverter dataConverter) {
     super(response.getWorkflowExecutionInfo(), dataConverter);
+    this.dataConverter = dataConverter;
     this.response = response;
   }
 
+  /** Get the fixed summary for this workflow execution. */
+  @Experimental
+  @Nullable
+  public String getStaticSummary() {
```

Comment:
Might consider memoizing this (e.g. backed by an `AtomicReference`). If not, at least worth clarifying in the javadoc each invocation does data conversion.